diagnostic.md

```diff
@@ -51,13 +59,17 @@ Write the `curl` request you'd use to delete a single donut, using whatever ID
 you'd like.
 
 ```sh
-# your answer here
+curl --request DELETE http://www.example.com/donuts/3
 ```
 
 Write an AJAX request to delete a single donut from the server.
 
 ```js
-let deleteDonut = /* your answer here */;
+let deleteDonut = function() {
+  $.ajax({
+    url: 'http://www.example.com/donuts/3',
+    type: 'DELETE'
+  });
```

Comment:
Ditto - without handlers, not only can you not operate on data that comes back, you can't tell your program to wait until the request finishes before doing something.,